script/room.js

```diff
@@ -797,6 +797,7 @@ var Room = {
 				$SM.set('stores["'+k+'"]', 0);
 			}
 			
+			lk = _(k);
```

Comment:
Needs `var` or you're polluting the global scope. Polluting the global scope is bad.